src/networking.c

```diff
@@ -2846,7 +2851,10 @@ sds catClientInfoString(sds s, client *client) {
         " redir=%I", (client->flags & CLIENT_TRACKING) ? (long long) client->client_tracking_redirection : -1,
         " resp=%i", client->resp,
         " lib-name=%s", client->lib_name ? (char*)client->lib_name->ptr : "",
-        " lib-ver=%s", client->lib_ver ? (char*)client->lib_ver->ptr : ""));
+        " lib-ver=%s", client->lib_ver ? (char*)client->lib_ver->ptr : "",
+        " tot-input=%U", client->net_input_bytes,
+        " tot-output=%U", client->net_output_bytes,
+        " cmd-proc=%U", client->commands_processed));
```

Comment:
Seems a little bit more consistent of a naming.
```suggestion
        " tot-cmds=%U", client->commands_processed));
```